apps/server/src/modules/me/api/mapper/me.response.mapper.ts

```diff
@@ -62,26 +62,26 @@ export class MeResponseMapper {
 		return userResponse;
 	}
 
-	private static mapUserRoles(user: User): MeRolesReponse[] {
+	private static mapUserRoles(user: User): MeRoleResponse[] {
 		const roles = user.getRoles();
 		const rolesResponse = roles.map((role) => MeResponseMapper.mapRole(role));
 
 		return rolesResponse;
 	}
 
-	private static mapRole(role: Role): MeRolesReponse {
-		const roleResponse = new MeRolesReponse({
+	private static mapRole(role: Role): MeRoleResponse {
+		const roleResponse = new MeRoleResponse({
 			id: role.id,
 			name: role.name,
 		});
 
 		return roleResponse;
 	}
 
-	private static mapPermissions(user: User): string[] {
-		const permissionStrings = user.resolvePermissions();
+	private static mapPermissions(permissions: Set<string>): string[] {
```

Comment:
can you use instead of a general string the actual permissions list that we have? (I think that is an enum?)
same for mapToResponse above

Reply:
There is a Permission enum and I also wanted to use it, but `user.resolvePermissions()` returns `string[]` and changing that would lead to a lot of other changes. I think this should be refactored in another PR. I'll take it with me and think about creating a ticket. Fine?